marlin/src/lib.rs

```diff
@@ -136,8 +136,11 @@ impl<F: PrimeField, PC: PolynomialCommitment<F>, D: Digest> Marlin<F, PC, D> {
         }
 
         let coeff_support = AHPForR1CS::get_degree_bounds::<C>(&index.index_info);
+        // Marlin only needs degree 2 random polynomials
+        let supported_hiding_bound = 2;
```

Comment:
degree of the random polynomial is 2, but the hiding bound is 1; the translation should be handled at the PC layer, no?

Reply:
Good point